### PR TITLE
Added documentation for alert configs

### PR DESCRIPTION
**Pull request by prashanth26**
Documentation for https://github.com/SimpleDataLabsInc/prophecy/pull/14019

### CODE REVIEW

File: docs/architecture/deployment/private-saas/configure-alerts.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+  },
+  "suspensionWindowinMinutes": 60
+}
+```
+
+### Supported Configuration Variables
+
+| Configuration variable name             | Description                                                                                                                                                                                                                  | Default value                                                               |
+| --------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------- |
+| `alertConfigs.maxAllowedOccuranceCount` | Number of consequetive occurences to hit before triggering an alert (warning/critical). Every intervel defaults to 30s.                                                                                                      | `20 (or) 10mins` for CPU/Memory, `3 (or) 1.5mins` for Disk Usage/File Count |
```
Comment (vaishali-prophecy) [inline docs/architecture/deployment/private-saas/configure-alerts.md:100]: Typos:
- consecutive occurrences
- interval

File: docs/architecture/deployment/private-saas/configure-alerts.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+| `alertConfigs.metricsType`              | The metric type tracked                                                                                                                                                                                                      | CPU_USAGE, DISK_USAGE, FILE_COUNT, MEMORY_USAGE                             |
+| `alertConfigs.thresholdValue`           | The threshold value beyond which alert is triggerred. This is a fractional value between 0 and 1                                                                                                                             | `0.8` for MEMORY_USAGE and `0.95` for others                                |
```
Comment (vaishali-prophecy) [inline docs/architecture/deployment/private-saas/configure-alerts.md:102]: *triggered

File: docs/architecture/deployment/private-saas/configure-alerts.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+| `enableAlerts`                          | Enabling of alerting system                                                                                                                                                                                                  | `true`                                                                      |
+| `notificationEmailIDs`                  | List of emails to notify in case of hitting alerts, this list is comma seperated                                                                                                                                             | `[]`                                                                        |
```
Comment (vaishali-prophecy) [inline docs/architecture/deployment/private-saas/configure-alerts.md:104]: *separated

File: docs/architecture/deployment/private-saas/configure-alerts.md [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+| `reAlertIntervelinMinutes`              | How often should triggerred alerts resend the email in minutes                                                                                                                                                               | `120`                                                                       |
```
Comment (vaishali-prophecy) [inline docs/architecture/deployment/private-saas/configure-alerts.md:105]: reAlertIntervalinMinutes

**Review by vaishali-prophecy — APPROVED** [review]

File: docs/architecture/deployment/private-saas/configure-alerts.md (hunk 4 above)
Comment (prashanth26) [reply docs/architecture/deployment/private-saas/configure-alerts.md]: Fixed here - https://github.com/SimpleDataLabsInc/prophecy-docs/pull/438/commits/21b9299875daa1d18ba705715b0dfc3f323879fe

**Review by alexanderahn — APPROVED** [review]

**Review by manjunathprophecy — APPROVED** [review]

**Pull request merged**